orchagent/portsorch.cpp

```diff
@@ -5252,8 +5252,22 @@ void PortsOrch::doTask(NotificationConsumer &consumer)
                 sai_uint32_t speed;
                 if (getPortOperSpeed(port, speed))
                 {
-                    SWSS_LOG_NOTICE("%s oper speed is %d", port.m_alias.c_str(), speed);
-                    updateDbPortOperSpeed(port, speed);
+                    if (speed != 0) 
```

Comment:
Should we just return false from `getPortOperSpeed` if speed is 0. Just a thought?